crates/op-rbuilder/src/builders/builder_tx.rs

```diff
@@ -167,3 +172,100 @@ pub trait BuilderTransactions<ExtraCtx: Debug + Default = ()>: Debug {
         Ok(simulation_state)
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct BuilderTxBase {
```

Comment:
```suggestion
pub(crate) struct BuilderTxBase {
```

unless it's supposed to be part of the public API?